packages/core/src/http-client.ts

```diff
@@ -60,6 +60,7 @@ export async function executeRequest<D extends BaseLlmParameters>(
   requestConfig?: CustomRequestConfig
```

Comment:
not directly related to your change, but what do you think about changing the type "D" to "Data" or something more speaking?